data-plane/receiver/src/main/java/dev/knative/eventing/kafka/broker/receiver/RequestToRecordMapper.java

```diff
@@ -0,0 +1,19 @@
+package dev.knative.eventing.kafka.broker.receiver;
+
+import io.vertx.core.Future;
+import io.vertx.core.http.HttpServerRequest;
+import io.vertx.kafka.client.producer.KafkaProducerRecord;
+import java.util.function.Function;
+
+public interface RequestToRecordMapper<K, V> extends
```

Comment:
Annotate with `@FunctionalInterface`, it could be useful in future

Reply:
Done